swap_meet/vendor.py

```diff
@@ -1,2 +1,142 @@
+        Swaps indicated items from inventory of two vendors.
+
+            Parameters:
+                    vendor: Another instance of Vendor 
+                    my_item: Item object that the current instance of Vendor gives
+                    their_item: Item object that the current instance of Vendor receives.
+            
+            Returns:
+                    False if Item is not in the inventory of either Vendor object
+                    True if swap is successful.
+        """
+        if not self.remove(my_item):
+            return False
+        elif not vendor.remove(their_item):
+            self.add(my_item)
+            return False
+        else:
+            self.remove(my_item)
+            vendor.add(my_item)
+
+            vendor.remove(their_item)
+            self.add(their_item)
+            return True
+    
+    def swap_first_item(self, vendor):
+        """
+        Swaps the first item from inventory of two vendors. 
+
+            Parameter: 
+                    vendor: Another instance of Vendor 
+            
+            Returns:
+                    False if vendor inventory is empty. 
+                    True if swap is successful. 
+        """
+        if len(self.inventory) == 0 or len(vendor.inventory) == 0:
+            return False
+        self.swap_items(vendor, self.inventory[0], vendor.inventory[0])
+        return True
+
+    def get_best_by_category(self, category):
+        """
+        Gets item in best condition from indicated category. 
+            
+            Parameter: 
+                    category(str): Name of category
+            
+            Returns:
+                    None if there are no items in inventory from indicated category. 
+                    Item object with highest condition within inidcated category. 
+        """
+        items_in_category = self.get_by_category(category)
+        if len(items_in_category) == 0:
+            return None
+        else:
+            return max(items_in_category, key = lambda item: item.condition)
+
+    def swap_best_by_category(self, other, my_priority, their_priority):
+        """
+        Swaps the best item in a specified category from the invetory of two vendors.
+
+            Parameters:
+                    other: Another instance of Vendor 
+                    my_priority(str): Name of category current instance of Vendor wants to receive 
+                    their_priority(str): Name of category other instance of Vendor wants to receive
+            Returns:
+                    False if there are no items that match the indicated categories. 
+                    True if swap is successful. 
+        """
+        my_best_item = self.get_best_by_category(their_priority)
+        their_best_item = other.get_best_by_category(my_priority)
+
+        if not my_best_item or not their_best_item:
+            return False
+        
+        self.swap_items(other, my_best_item, their_best_item)
+        return True
```

Comment:
This is very nitpicky and definitely not a big deal, but check out lines 118-121. You have an `else` statement there, but not here. Whatever you choose (`else` or no `else`) stay consistent. Try to style your methods all the same